platforms/ide/tooling-api/src/main/java/org/gradle/tooling/events/problems/internal/DefaultSingleProblemEvent.java

```diff
@@ -78,7 +82,19 @@ public Details getDetails() {
 
     @Override
     public List<Location> getLocations() {
-        return locations;
+        return ImmutableList
+            .<Location>builder()
+            .addAll(originLocation)
+            .addAll(attributedLocations)
+            .build();
+    }
+
+    public List<Location> getOriginLocation() {
```

Comment:
❌ Should be available on `SingleProblemEvent` interface.

Reply:
Pulled up